src/graphql/schema/User/resolvers.js

```diff
@@ -20,7 +20,7 @@ export default {
       const user = await User.findOne({ email });
       let valid = false;
       if (user) {
-        valid = bcrypt.compare(password, user.password);
+        valid = true === (await bcrypt.compare(password, user.password));
```

Comment:
I just wanted to be really sure that we receive a boolean of value true back because otherwise any truthy value (like an unresolved or probably even rejected promises) would authorize the user. Probably overkill but i thought: no harm no foul.